packages/office-ui-fabric-react/src/components/Icon/ImageIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+import { IImageIconProps } from './Icon.types';
+import { Image } from '../Image/Image';
+import { css, getNativeProps, htmlElementProperties } from '../../Utilities';
+import { classNames, MS_ICON } from './Icon.styles';
+
+export const ImageIcon: React.StatelessComponent<IImageIconProps> = props => {
+  const { className, imageProps } = props;
+
+  const nativeProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(props, htmlElementProperties);
+
+  const containerProps = props['aria-label']
+    ? {}
```

Comment:
Same as with `FontIcon`, shouldn't `'aria-label': props['aria-label']` be set here?

Reply:
It will get passed through automatically as one of the native props.